looper/command_models/arguments.py

```diff
@@ -0,0 +1,168 @@
+        and it's a bit tedious.
+
+        """
+        c = copy(self)
+        _, default_value = self.default
+        c.default = default_value
+        return c
+
+
+class ArgumentEnum(enum.Enum):
+    """
+    Lists all available arguments
+
+    Having a single "repository" of arguments allows us to re-use them easily across different commands.
+
+    TODO: not sure whether an enum is the ideal data structure for that
+    """
+
+    IGNORE_FLAGS = Argument(
+        name="ignore_flags",
+        default=(bool, False),
+        description="Ignore run status flags",
+    )
+    TIME_DELAY = Argument(
+        name="time_delay",
+        default=(int, 0),
+        description="Time delay in seconds between job submissions (min: 0, max: 30)",
+    )
+    DRY_RUN = Argument(
+        name="dry_run", default=(bool, False), description="Don't actually submit jobs"
+    )
+    COMMAND_EXTRA = Argument(
+        name="command_extra",
+        default=(str, ""),
+        description="String to append to every command",
+    )
+    COMMAND_EXTRA_OVERRIDE = Argument(
+        name="command_extra_override",
+        default=(str, ""),
+        description="Same as command-extra, but overrides values in PEP",
+    )
+    LUMP = Argument(
+        name="lump",
+        default=(float, None),
+        description="Total input file size (GB) to batch into one job",
+    )
+    LUMPN = Argument(
+        name="lumpn",
+        default=(int, None),
+        description="Number of commands to batch into one job",
+    )
+    LIMIT = Argument(
+        name="limit", default=(int, None), description="Limit to n samples"
+    )
+    SKIP = Argument(
+        name="skip", default=(int, None), description="Skip samples by numerical index"
+    )
+    CONFIG_FILE = Argument(
+        name="config_file",
+        default=(str, None),
+        description="Project configuration file",
+    )
+    LOOPER_CONFIG = Argument(
+        name="looper_config",
+        default=(str, None),
+        description="Looper configuration file (YAML)",
+    )
+    SETTINGS = Argument(
+        name="settings",
+        default=(str, ""),
+        description="Path to a YAML settings file with compute settings",
+    )
+    PEP_CONFIG = Argument(
+        name="pep_config",
+        default=(str, None),
+        description="PEP configuration file",
+    )
+    OUTPUT_DIR = Argument(
+        name="output_dir",
+        default=(str, None),
+        description="Output directory",
+    )
+    SAMPLE_PIPELINE_INTERFACES = Argument(
+        name="sample_pipeline_interfaces",
+        default=(list, []),
+        description="Paths to looper sample config files",
+    )
+    PROJECT_PIPELINE_INTERFACES = Argument(
+        name="project_pipeline_interfaces",
+        default=(list, []),
+        description="Paths to looper project config files",
+    )
+    AMEND = Argument(
+        name="amend", default=(list, []), description="List of amendments to activate"
```

Comment:
slight preference for a plural noun when the value can be multiple, but NBD if it's difficult to do

Reply:
Similar to my other comment above: argument names and descriptions were copied verbatim from `cli_looper.py` to ensure compatibility, and ideally we would fix them in that file, and then in this one. Another option would be to draw argument names and descriptions in `cli_looper.py` from this file and regard it as the ground truth from now on.